### ISSUE TITLE
[JENKINS-57365] MsBuildParser reports warnings if the build project contains "info"

### ISSUE THREAD

**Issue by jenkins-infra-bot** [jenkinsci/analysis-model#1343]
<p>The warning analysis reports a false positive if e.g. the build project name contains the string "info".</p>

<p>For example, the line below is matched by the regexp parser:</p>

<pre>
311>------ Build started: Project: sysinfo, Configuration: Debug Win32 ------</pre>
<p> <br/>
 This obviously leads to an error during the report generation:</p>

<pre>
[DEBUG] [-ERROR-] Can't resolve absolute paths <span class="code-keyword">for</span> some files:
[DEBUG] [-ERROR-] - C:/jenkins/workspace/acecode-win/build/------ Build started: Project
[DEBUG] [-ERROR-] Can't create fingerprints <span class="code-keyword">for</span> some files:
[DEBUG] [-ERROR-] - <span class="code-quote">'C:/jenkins/workspace/acecode-win/build/------ Build started: Project'</span>, IO exception has been thrown: java.nio.file.InvalidPathException: Illegal <span class="code-object">char</span> <:> at index 59: C:/jenkins/workspace/acecode-win/build/------ Build started: Project
</pre>
<p> </p>

---
<details><summary><i>Originally reported by <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=stephansenkbeil">stephansenkbeil</a>, imported from: <a class="original-jira-link" href="https://issues.jenkins.io/browse/JENKINS-57365" target="_blank">MsBuildParser reports warnings if the build project contains "info"</a></i></summary>
<i><ul>
<li><b>assignee</b>: <a href="https://issues.jenkins.io/secure/ViewProfile.jspa?name=stephansenkbeil">stephansenkbeil</a>
<li><b>status</b>: Resolved
<li><b>priority</b>: Minor
<li><b>component(s)</b>: analysis-model
<li><b>label(s)</b>: help-wanted, newbie-friendly
<li><b>resolution</b>: Fixed
<li><b>resolved</b>: 2019-05-14T09:59:37+00:00
<li><b>votes</b>: 0
<li><b>watchers</b>: 2
<li><b>imported</b>: 20260108-221235
</ul></i>
<details><summary>Raw content of original issue</summary>

<pre>
<p>The warning analysis reports a false positive if e.g. the build project name contains the string "info".</p>

<p>For example, the line below is matched by the regexp parser:</p>
<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java">311&gt;------ Build started: Project: sysinfo, Configuration: Debug Win32 ------</pre>
</div></div>
<p> 
 This obviously leads to an error during the report generation:</p>
<div class="code panel" style="border-width: 1px;"><div class="codeContent panelContent">
<pre class="code-java">[DEBUG] [-ERROR-] Can't resolve absolute paths <span class="code-keyword">for</span> some files:
[DEBUG] [-ERROR-] - C:/jenkins/workspace/acecode-win/build/------ Build started: Project
[DEBUG] [-ERROR-] Can't create fingerprints <span class="code-keyword">for</span> some files:
[DEBUG] [-ERROR-] - <span class="code-quote">'C:/jenkins/workspace/acecode-win/build/------ Build started: Project'</span>, IO exception has been thrown: java.nio.file.InvalidPathException: Illegal <span class="code-object">char</span> &lt;:&gt; at index 59: C:/jenkins/workspace/acecode-win/build/------ Build started: Project
</pre>
</div></div>
<p> </p></pre>
</details>
</details>
<details><summary><i>environment</i></summary>

```
Jenkins 2.176<br/>
Analysis Model API Plug-in 5.0.0<br/>
Warnings Next Generation Plugin 5.0.0
```
</details>
